core/src/main/java/eu/tsystems/mms/tic/testframework/utils/SourceUtils.java

```diff
@@ -103,6 +103,7 @@ private static Optional<StackTraceElement> traceStackTraceElement(Throwable thro
 
     private static Optional<File> findClassFile(String className) {
         String sourceRoot = Report.Properties.SOURCE_ROOT.asString();
+        if(className.contains("$")) className = className.substring(0,className.indexOf("$")); //this is needed for inner classes
```

Comment:
Please always use brackets `{...}` and format code with IntelliJ.
The rest is fine.

Reply:
Done